aws-test/tests/aws_config_configuration_recorder/variables.tf

```diff
@@ -7,13 +7,13 @@ variable "resource_name" {
 
 variable "aws_profile" {
   type        = string
-  default     = "integration-tests"
+  default     = "default"
   description = "AWS credentials profile used for the test. Default is to use the default profile."
 }
 
 variable "aws_region" {
   type        = string
-  default     = "us-east-1"
+  default     = "us-west-2"
```

Comment:
Did we need to change this? What do other tests use?

Reply:
Ahh, My bad reverted back